### ISSUE TITLE
More code and tutorial resources

### ISSUE THREAD

**Issue by gabrielziegler3** [facebookresearch/fastMRI#127]
Hi, 
I am an undergraduate student applying a lot of the code provided in fastMRI in my bachelor thesis, but some features I find a bit hard to see how they properly work because I can't see many open-sourced code using them. Would it be possible to gather some open-source models and data transformation steps used with the fastMRI dataset so it is clearer how things work in practice? Or maybe point out some more examples currently available. The notebook demo is quite helpful, but also very limited. Would love to see more content like that available.

Thank you

**Comment by mmuckley** [comment]
I think this is reasonable. I don't think I'm able to put something together at the moment, but if anyone wants to file a PR to explain more of the repository I'd be happy to review it.

**Comment by gabrielziegler3** [comment]
I can't currently work on that now, but whenever I get more notebooks using different FastMRI methods, I'd be happy to contribute.

**Comment by finger1517** [comment]
I recently have read the GRAPPANET paper, but there is no code for how g factor is trained, and I could not find the source, anyone knows that?

**Comment by mmuckley** [comment]
@finger1517 
I don't think this one was open-sourced. I just opened Issue #136 in case someone from the community would like to contribute it.
